tests/sentry/api/endpoints/test_organization_plugins_configs.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+from django.conf import settings
+
+from sentry.plugins.base import plugins
+from sentry.testutils import APITestCase
+from sentry.runner.initializer import register_plugins, unregister_plugins
+
+
+class OrganizationPluginsTest(APITestCase):
+    def setUp(self):
+
+        register_plugins(settings, raise_on_plugin_load_failure=True)
```

Comment:
Does calling this multiple times result in the `SENTRY_DEFAULT_INTEGRATIONS ` being registered multiple times and never unregistered?